src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -140,6 +140,11 @@ class OptionsResolver implements Options
      */
     private $prototypeIndex;
 
+    /**
+     * @var bool
```

Comment:
should be replaced by a native type

Reply:
do you mean make like this?
private bool $ignoreUndefined = false;

if yes, there is any property with native type declaration in the class.
is it ok to do so?

Reply:
done